packages/plugins/rubygem-foreman_azure_rm/rubygem-foreman_azure_rm.spec

```diff
@@ -97,6 +97,7 @@ cp -a .%{gem_dir}/* \
 %license %{gem_instdir}/LICENSE
 %{gem_instdir}/app
 %{gem_instdir}/config
+%{gem_instdir}/db
```

Comment:
@evgeni @zjhuntin I have added `%{gem_instdir}/db` manually as the script did not generate this line by itself. Added this because I've introduced the `db` directory in the plugin. [foreman_azure_rm/73](https://github.com/theforeman/foreman_azure_rm/pull/73) and [foreman_azure_rm/75](https://github.com/theforeman/foreman_azure_rm/pull/75). Let me know if the `db` directory should mentioned at any other place too.